src/components/ListItemIcon/ListItemIcon.vue

```diff
@@ -93,64 +92,101 @@ import Avatar from '../Avatar'
 import Highlight from '../Highlight'
 import { userStatus } from '../../mixins'
 
+const margin = 6
+
 export default {
-	name: 'AvatarSelectOption',
+	name: 'ListItemIcon',
+
 	components: {
 		Avatar,
 		Highlight,
 	},
 	mixins: [userStatus],
+
 	props: {
 		/**
-		 * Secondary optional line
+		 * Default first line text
 		 */
-		desc: {
+		title: {
 			type: String,
-			default: '',
+			required: true,
 		},
+
 		/**
-		 * Default text
+		 * Secondary optional line
 		 */
-		displayName: {
+		subtitle: {
 			type: String,
-			required: true,
+			default: '',
 		},
+
 		/**
 		 * Icon class to be displayed at the end of the component
 		 */
 		icon: {
 			type: String,
 			default: '',
 		},
+
 		/**
-		 * See the [Avatar](#Avatar) user prop
+		 * Search within the highlight of title/subtitle
 		 */
-		user: {
+		search: {
 			type: String,
 			default: '',
 		},
+
 		/**
-		 * The status of the user
+		 * Set a size in px for the height
 		 */
-		status: {
-			type: Object,
+		size: {
```

Comment:
```suggestion
		avatarSize: {
```

Reply:
Or maybe `height`? Size sounds a bit vague to me

Reply:
Well, it is a square, so it would also span width :)

Reply:
Done! Thanks :)